vkext/vkext-rpc-tl-serialization.cpp

```diff
@@ -1356,19 +1432,17 @@ void vk_memcache_query_result1(INTERNAL_FUNCTION_PARAMETERS) {
   auto *T = reinterpret_cast<tl_tree *>(q->extra);
   INC_REF (T);
   if (do_rpc_get_and_parse(qid, timeout - precise_now) < 0) {
-    zval *r = make_query_result_or_error(NULL, "Response not found, probably timed out",
-                                         TL_ERROR_RESPONSE_NOT_FOUND);
+    zval *r = make_query_result_or_error(NULL, {TL_ERROR_RESPONSE_NOT_FOUND, "Response not found, probably timed out"});
```

Comment:
may be refactor all NULL to nullptr?

Reply:
I think it's better to do it in separate PR